core/server/data/schema/fixtures/utils.js

```diff
@@ -58,20 +57,6 @@ matchFunc = function matchFunc(match, key, value) {
     };
 };
 
-matchObj = function matchObj(match, item) {
```

Comment:
Will need to come back to this one as I'm not sure the matchObj is equivalent to matchFunc. Might need a unit test or some other way to verify it's not breaking anything.

Reply:
@daniellockyer  I think it would be safer to keep this method as is and pass an object into the new `where` method instead of a function. Bookshelf change that deprecated the old interface even has object notation documented as a preferred method - https://github.com/bookshelf/bookshelf/pull/2001/files#diff-a3644fc6e558da2bde71112d24ec75baR386-R407.

Was trying to figure out a good way to validate this area, but our test coverage is quite limited to what is checkes and isn't covering these matchObj/matchFunc functions at all. How did you verify this change didn't break anything apart from existing unit tests?

Reply:
You're absolutely correct. I'll go back and fix this, and ping you when I'm ready again.